docs/xsl/common.xsl

```diff
@@ -34,7 +34,7 @@
   <xsl:param name="table.cell.border.color" select="'#747474'"/>
   <xsl:param name="table.cell.border.style" select="'solid'"/>
   <xsl:param name="table.cell.border.thickness" select="'1px'"/>
-  <xsl:param name="table.frame.border.color" select="'#FF8080'"/>
+  <xsl:param name="table.frame.border.color" select="'#108768'"/>
```

Comment:
Is the idea here to switch from red to green borders? If you want the JanusGraph green color, based on my research it's #1b8669; see [`style.css`](https://github.com/JanusGraph/janusgraph.org/blob/master/style.css#L18) in the janusgraph.org repo.